content/instructor/quick-resources/index.mdx

```diff
@@ -0,0 +1,51 @@
+---
+date: 2020-04-13
+title: 'Quick Resources Page'
+description: "Getting ready to record? Find the information you need on one page!"
+categories: ['instructor']
+published: true
+shareImage: 'https://res.cloudinary.com/dg3gyk0gu/image/upload/v1571260698/og-image-assets/share_image_screencasting.png'
+redirect_from:
+
+---
+This **Quick Resources Page** compiles technical specifications that you might need to reference as you begin recording your lessons!
+
+⚠️ First, check out this comprehensive course by John to get started: [Record Badass Screencasts for egghead.io](https://egghead.io/courses/record-badass-screencasts-for-egghead-io)
+
+|     |     |      |
+|:-----:|:-----:|:------:|
+|[Desktop Environment](#desktop-environment)|[Audio Environment](#audio-environment)|[Recording](#recording)|
+|[Editing Videos](#editing-videos)|[Exporting Videos](#exporting-videos)|
+
+### Desktop Environment
+- Use large fonts and high-contrast color schemes
+- Record fullscreen at **16:9 aspect ratio**
+- **HiDPI** mode on monitor: **1280x720 pixel** resolution
+    1. [How on Windows](screencasting/screen-setup#dpi-scaling-on-windows)
+    2. [How on MacOS](screencasting/screen-setup#access-hidpi-mode-on-macos)
+- Use **2:1 ratio** of **code:example**:
+![](https://lh5.googleusercontent.com/yFu1UkYEzFM_zbwuf6kxZU5WuUwc_zhkaJY6uDSwk32Tega5Mj6rNukMByPiNZseZVyQgxPcRVsB07yh5R6IkiBUHnkm4mtAB9XOGJ4wt3qX0ZYcKw1kTVp0VJ-qNb9wJyctddH4)
```

Comment:
This image looks a little awkward on the page with the whitespace within the image

![Screen Shot 2020-04-23 at 3 20 04 PM](https://user-images.githubusercontent.com/6188161/80140467-fb2e3a00-8575-11ea-8f61-8e6800c5d1cf.png)